l10n_fr_hr_check_ssnid/views/hr_employee.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2018 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo>
+
+
+<record id="view_employee_form" model="ir.ui.view">
+    <field name="model">hr.employee</field>
+    <field name="inherit_id" ref="hr.view_employee_form" />
+    <field name="arch" type="xml">
+        <field name="identification_id" position="after">
+            <field name="ssnid" groups="hr.group_hr_user"/>
```

Comment:
Hehe. always funny to have to add in OCA views, fields created by Odoo SA, but never used.